eth/tracers/native/call_flat.go

```diff
@@ -54,6 +54,7 @@ var parityErrorMapping = map[string]string{
 }
 
 var parityErrorMappingStartingWith = map[string]string{
+	"out of gas:":     "Out of gas", // convert OOG wrapped errors, eg `out of gas: not enough gas for reentrancy sentry`
```

Comment:
Is the casing correct here? Just asking to be sure.

Reply:
yes, actually they come from https://github.com/ethereum/go-ethereum/blob/fad7e74a1b/core/vm/interpreter.go#L279-L281

```go
			cost += dynamicCost // for tracing
			if err != nil {
				return nil, fmt.Errorf("%w: %v", ErrOutOfGas, err)
			}
```

Reply:
Question is if other clients return "Out of gas" or "out of gas". I thought that's why you're changing this.

Reply:
Still would like an answer on this.